shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dml/OracleInsertStatement.java

```diff
@@ -17,13 +17,41 @@
 
 package org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.dml;
 
+import lombok.Setter;
 import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.subquery.SubquerySegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.InsertMultiTableElementSegment;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.dml.InsertStatement;
 import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
 
+import java.util.Optional;
+
 /**
  * Oracle insert statement.
  */
+@Setter
 @ToString
 public final class OracleInsertStatement extends InsertStatement implements OracleStatement {
+    
+    private SubquerySegment selectSubquery;
```

Comment:
Why add selectSubquery and insertMultiTableElementSegment?

Reply:
In an `INSERT` statement, with the help of `insertIntoClause` we can add table elements such as `SimpleTableSegment`, `SubquerySegment`. Those are available in `InsertStatement`. But in Oracle's `insertSingleTable` and `insertMultiTable` clause, it expects a `SubquerySegment` after `insertIntoClause`. I think this is specific to Oracle insert statement.

![insertSingleTable](https://user-images.githubusercontent.com/48581379/123531196-1e2b1f80-d720-11eb-9f0c-a305b285eafb.png)

![insertMultiTable](https://user-images.githubusercontent.com/48581379/123531197-22573d00-d720-11eb-929f-b3c61c90ddc3.png)

Reply:
I added `insertMultiTableElementSegment` to support multitable inserts.